### ISSUE TITLE
[Accessibility][Screen Readers-Filter Dialog] : Screen Reader does not announce the search results of filter dialogs to the users.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2055]
**User Experience** 
If a screen reader will not provide any information about the on going activities on the screen then screen reader dependent users will never get to know about the what search results are appearing on screen.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and set the zoom at 400%.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the filter dialog edit field.
4. Type anything in the search edit field of 'Filter Dialogs'.
5. Observe the issue.

**Actual:** 
The narrator didn't inform about the search result of the 'Filter Dialogs' search edit field.

**Expected:**
As soon as user hits enter button after typing any dialog name in filter dialog search box, search results appears on screen. Screen reader should announce something like "<Dialog name> <no. of search results> search results have found. Press down arrow key to navigate the search results".

**Note:**
Same issue observed with NVDA.

**MAS Impacted:** [MAS4.2.12](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b19c05bd-d82f-4d6d-a8e5-7081bf5c2dba})

**Attachment:**
[MAS4.2.12_Search result not announced.zip](https://github.com/microsoft/BotFramework-Composer/files/4240981/MAS4.2.12_Search.result.not.announced.zip)

**Comment by corinagum** [comment]
Hey @tonyanziano, You created the filter-search component (what's the proper name??) in Emulator, right? Perhaps you have input / advice for this similar component when making accessible in Composer. Also tagging you as a resource to the developer who works on this :P

**Comment by ashish315** [comment]
@cwhitten As checked issue still repro.

**Comment by liweitian** [comment]
@ashish315 
[narrator_filter_dialog.zip](https://github.com/microsoft/BotFramework-Composer/files/4562536/narrator_filter_dialog.zip)
I think the narrator works as expected. Is there anything I missed here?

**Comment by ashish315** [comment]
@liweitian agree with you. the issue is resolved now. So, we are closing the issue.
